odxtools/cli/find.py

```diff
@@ -131,4 +140,4 @@ def run(args: argparse.Namespace) -> None:
         service_names=args.service_names,
         print_params=not args.no_details,
         allow_unknown_bit_lengths=args.relaxed_output,
-    )
+        print_objects=args.plumbing_output)
```

Comment:
keep same variable name across codebase
```suggestion
        plumbing_output=args.plumbing_output)
```